.github/workflows/build-docker.yml

```diff
@@ -115,5 +118,22 @@ jobs:
             export LDFLAGS=-Wl,--no-as-needed ${LDFLAGS}
           fi
           ./scripts/ci/test-examples.sh
+      - name: Upload test results
+        # Note: upload-v3 doesn't seem to pick up the paths: delete the "false" line below once we 
+        # upgrade the image
+        if: >-
+          ${{
+            false &&
+            always()
+            && (steps.test.outcome == 'success'
+                || steps.test.outcome == 'failure')
+            && (env.CMAKE_PRESET != 'ndebug-vecgeom')
```

Comment:
Why is this build excluded from the uploads?

Reply:
Hmm I think I was playing around with why the test was failing and said hey, maybe it's because it didn't have any google tests? But I think the real issue is the old uploader version. I'll delete this line.